user-guide/Advanced_Functionality/Databases/Indexing_database/OpenSearch_Database/Configuring_multiple_OpenSearch_clusters.md

```diff
@@ -32,7 +32,7 @@ To configure this setup, proceed as follows:
 
    - *PWD*: The password to connect to OpenSearch.
 
-   - *DB*: The prefix for the OpenSearch indexes.
+   - *DB*: The prefix for the OpenSearch indexes. The specified value should match across all the OpenSearch configurations in the *DBConfiguration.xml*
```

Comment:
DBConfiguration.xml is no longer used from DataMiner 10.4.0/10.4.2 onwards, so I'd like to change this as follows:

```suggestion
   - *DB*: The prefix for the OpenSearch indexes. The specified value must be the same for each OpenSearch cluster defined in *DB.xml*.
```